environment-api/src/main/java/com/sequenceiq/environment/api/doc/environment/EnvironmentModelDescription.java

```diff
@@ -148,6 +148,9 @@ public class EnvironmentModelDescription {
     public static final String ENVIRONMENT_DOMAIN_NAME = "The domain name that's has been generated for the environment.";
 
     public static final String AWS_DISK_ENCRYPTION_PARAMETERS = "AWS Disk encryption parameters";
+
+    public static final String NO_OUTBOUND_LOAD_BALANCER = "Flag that marks the request to not create outbound load balancer";
```

Comment:
Grammar fix: "... to not create **an** outbound load balancer"

Reply:
Fixed